### PR TITLE
Update FAQ-UseWasabi.md

### PR DESCRIPTION
**Pull request by BTCparadigm**
Trying to salvage work already done from #1094

i-am-once-again-asking-for-your-pull-request-approval-15667-1

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -9,34 +9,49 @@
...
+
 As with everything in Wasabi, you are required to label this new wallet.
-Make sure that you are precise so that you know what this wallet is for.
-The password is used to encrypt the private key (extracted from the extended private key) on the computer.
+Make sure that you are precise so that you know what this wallet is for and click `Continue`.
 
-:::danger Careful!!
-Without knowledge of the password, you CANNOT spend your bitcoin!!
-:::
+![Name a new wallet in Wasabi](/AddWalletName.png "Name a new wallet in Wasabi")
```
Comment (MarnixCroes) [inline docs/FAQ/FAQ-UseWasabi.md:23]:
```suggestion
![Name a new wallet in Wasabi](/AddWalletWalletName.png "Name a new wallet in Wasabi")
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -9,34 +9,49 @@
...
+
+Add a password. 
+It is used to encrypt the private key (extracted from the extended private key) on the computer and needed to open the wallet.
+
+![Add a password in Wasabi](/AddWalletPassword.png "Add a password in Wasabi")
```
Comment (MarnixCroes) [inline docs/FAQ/FAQ-UseWasabi.md:28]:
```suggestion
![Add a password in Wasabi](/AddWalletAddPassword.png "Add a password in Wasabi")
```

**Review by MarnixCroes — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md (hunk 2 above)
Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md:28]: `AddWalletPassword.png` doesn't exit

Comment (BTCparadigm) [reply docs/FAQ/FAQ-UseWasabi.md:28]: 4gdcznnjxyk71

Good job!

File: docs/FAQ/FAQ-UseWasabi.md [hunk 3]
```diff
@@ -9,34 +9,49 @@
...
 ::::details
 ### How do I generate a new wallet?
 
 You can [generate as many new wallets](/using-wasabi/WalletGeneration.md) as you'd like, for no extra cost and without asking for permission.
-Go to the `Wallet Manager` tab and the `Generate Wallet` menu.
+Click the `Add Wallet` button in the bottom left corner of the main view or in the searchbar.
+
+![Add new wallet in Wasabi](/AddWalletButton.png "Add a new wallet in Wasabi")
+
 As with everything in Wasabi, you are required to label this new wallet.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:20]: There is a step missing before this one, and is to chose/click `Create a new wallet`

File: docs/FAQ/FAQ-UseWasabi.md [hunk 4]
```diff
@@ -9,34 +9,49 @@
...
-Make sure that you are precise so that you know what this wallet is for.
-The password is used to encrypt the private key (extracted from the extended private key) on the computer.
+Make sure that you are precise so that you know what this wallet is for and click `Continue`.
 
-:::danger Careful!!
-Without knowledge of the password, you CANNOT spend your bitcoin!!
-:::
+![Name a new wallet in Wasabi](/AddWalletWalletName.png "Name a new wallet in Wasabi")
+
+Add a password.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:25]: This step is after confirming the recovery words now.

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by yahiheb** [comment]
One more thing, to keep the repository clean, one should fork and create branches on their own fork, then open a PR.

File: docs/FAQ/FAQ-UseWasabi.md [hunk 5]
```diff
@@ -9,34 +9,49 @@
...
-
 ::::details
 ### How do I generate a new wallet?
 
 You can [generate as many new wallets](/using-wasabi/WalletGeneration.md) as you'd like, for no extra cost and without asking for permission.
-Go to the `Wallet Manager` tab and the `Generate Wallet` menu.
+Click the `Add Wallet` button in the bottom left corner of the main view or in the searchbar.
+
+![Add new wallet in Wasabi](/AddWalletButton.png "Add a new wallet in Wasabi")
+
```
Comment (BTCparadigm) [inline docs/FAQ/FAQ-UseWasabi.md:19]:
```suggestion

Click `Create a new wallet`.

![Create a new wallet in Wasabi](/AddWallet.png "Create a new wallet in Wasabi")

```

File: docs/FAQ/FAQ-UseWasabi.md (hunk 4 above)
Comment (BTCparadigm) [reply docs/FAQ/FAQ-UseWasabi.md:40]: https://github.com/zkSNACKs/WasabiDoc/pull/1273/commits/d0ed14c8d8f73733f9966d73bc8029c0720e5eb0

File: docs/FAQ/FAQ-UseWasabi.md [hunk 6]
```diff
@@ -9,34 +9,53 @@
...
-![Wasabi Wallet recovery words](/AddWalletRecoveryWords.png "Wasabi Wallet recovery words")
+![Add a password in Wasabi](/AddWalletAddPassword.png "Add a password in Wasabi")
+
+Make sure that you properly back up and write down this password.
+In order to protect your backup, consider storing the password and recovery words in two different locations.
+Confirm the password and click `Continue`.
+
+:::danger Careful!!
+Without knowledge of the password, you CANNOT spend your bitcoin!!
+:::
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:53]:
```suggestion
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 7]
```diff
@@ -9,34 +9,53 @@
...
+
+Add a password. 
+It is used to encrypt the private key (extracted from the extended private key) on the computer and needed to open the wallet.
 Make sure, and triple-check that you have done a proper backup of BOTH the password AND the mnemonic.
 When you only have the recovery words, but not the password, then you will NOT be able to recover your wallet.
-In order to protect your paper backup, consider storing the password and recovery words in two different locations.
-You have successfully setup your wallet when you click `I wrote down my Recovery Words!`
 
-![Wasabi Wallet recovery words](/AddWalletRecoveryWords.png "Wasabi Wallet recovery words")
+![Add a password in Wasabi](/AddWalletAddPassword.png "Add a password in Wasabi")
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:45]:
```suggestion
![Add a password in Wasabi](/AddWalletAddPassword.png "Add a password in Wasabi")

:::danger Careful!!
Without knowledge of the password, you CANNOT spend your bitcoin!!
:::
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 8]
```diff
@@ -9,34 +9,55 @@
...
+
+![Name a new wallet in Wasabi](/AddWalletWalletName.png "Name a new wallet in Wasabi")
+
+Now you will see the 12 recovery words.
+This is the mnemonic seed that you should back up, together with the password you create in the next phase, in order to recover your wallet.
+When you've backed everything up, click `Continue`.
+
+![Wasabi Wallet recovery words](/AddWalletRecoveryWords.png "Wasabi Wallet recovery words")
+
+In this page you can do a quick check that you've written down your mnemonic seed in the correct order.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:36]:
```suggestion
In this page you should do a quick check that you've written down your mnemonic seed in the correct order. 
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 9]
```diff
@@ -9,34 +9,55 @@
...
+
+![Wasabi Wallet recovery words](/AddWalletRecoveryWords.png "Wasabi Wallet recovery words")
+
+In this page you can do a quick check that you've written down your mnemonic seed in the correct order. 
+Once you're done, click `Continue`.
+
+![Confirm recovery words](/AddWalletConfirmRecoveryWords.png "Confirm recovery words")
+
+Add a password. 
+It is used to encrypt the private key (extracted from the extended private key) on the computer and needed to open the wallet.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:42]:
```suggestion
It is used to encrypt the private key (extracted from the extended private key) on the computer and needed to open the wallet and to recovered it in the future.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 10]
```diff
@@ -9,34 +9,55 @@
...
+Make sure that you properly back up and write down this password.
+
+Confirm the password and click `Continue`.
+
+![Add a password in Wasabi](/AddWalletAddPassword.png "Add a password in Wasabi")
 
 :::danger Careful!!
 Without knowledge of the password, you CANNOT spend your bitcoin!!
 :::
+Make sure, and triple-check that you have done a proper backup of BOTH the password AND the mnemonic.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:52]:
```suggestion

Make sure, and triple-check that you have done a proper backup of BOTH the password AND the mnemonic.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 11]
```diff
@@ -9,34 +9,55 @@
...
+If you only have the recovery words, but not the password, then you will NOT be able to open your wallet or to recover it.
 
-Make sure that you properly back up and write down this password.
-Please also read and agree to the [terms and conditions, the privacy policy and the legal statement](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Legal/Assets/LegalDocumentsWw2.txt).
-Now you can continue with clicking `Generate`.
+In order to protect your backup, consider storing the password and recovery words in two different locations.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:55]:
```suggestion
In order to protect your backup, consider storing the password and recovery words in different locations.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 12]
```diff
@@ -9,34 +9,55 @@
...
 
-![Generate new wallet in Wasabi](/WalletManagerGenerateWallet.png "Generate new wallet in Wasabi")
+Now you get to choose your wallets coinjoin strategy.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:57]:
```suggestion
Now you get to choose your wallets coinjoin strategy.
Select one and click `Continue`.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]

**Pull request merged**